index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Jake's Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <div id="head_logo">Jake Felice</div>
+        <div id="menu-button"></div>
+    </header>
+    <section id="intro">
+        <div id="intro-content">

```

Comment:
- [ ] Please add the background image for this section to match the given design.

![image](https://user-images.githubusercontent.com/68796232/158659951-967d0e53-0c75-4d33-9f60-68974e6ea364.png) 
![image](https://user-images.githubusercontent.com/68796232/158660060-a483553a-0cef-470c-9dcc-5daef7c56525.png)